build/Dockerfile

```diff
@@ -4,13 +4,16 @@
 FROM alpine:3.15
 LABEL maintainer="offen <hioffen@posteo.de>"
 
+ARG TARGETARCH
+ARG TARGETVARIANT
+
 RUN addgroup -g 10001 -S offen \
 	&& adduser -u 10000 -S -G offen -h /home/offen offen
 RUN apk add -U --no-cache ca-certificates libcap tini bind-tools
 
-COPY ./bin/offen-linux-amd64 /opt/offen/offen-linux-amd64
-RUN setcap CAP_NET_BIND_SERVICE=+eip /opt/offen/offen-linux-amd64
-RUN ln -s /opt/offen/offen-linux-amd64 /sbin/offen
+COPY ./bin/offen-linux-$TARGETARCH${TARGETVARIANT:+-$TARGETVARIANT} /opt/offen/offen-linux-$TARGETARCH
```

Comment:
I wonder if we should always use the full name for the sake of consistency, e.g.:

```
COPY ./bin/offen-linux-$TARGETARCH${TARGETVARIANT:+-$TARGETVARIANT} /opt/offen/offen-linux-$TARGETARCH${TARGETVARIANT:+-$TARGETVARIANT}
```

The same applies for the two lines below.

Reply:
I was having the same thought when I pushed it. Since you also brought it up I'll add the change.